spec/features/admin/admin_sees_order_details_spec.rb

```diff
@@ -59,7 +59,7 @@
     click_on "Order Dashboard"
   end
 
-  xscenario 'can view specific order details' do
+  scenario 'can view specific order details' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.